config/config.yaml

```diff
@@ -6,6 +6,7 @@ kraken:
     signal_state: RUN                                      # Will wait for the RUN signal when set to PAUSE before running the scenarios, refer docs/signal.md for more details
     signal_address: 0.0.0.0                                # Signal listening address
     port: 8081                                             # Signal port
+    pod_recovery_time: 30                                  # pod recovery time after kill pod scenario.
```

Comment:
Pod scenario config might be a better place for setting this I guess. Thoughts?

Reply:
+1, we might want to set different times based on different scenarios